staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1/types.go

```diff
@@ -654,13 +654,13 @@ type CDICloneStrategy string
 
 const (
 	// CloneStrategyHostAssisted specifies slower, host-assisted copy
-	CloneStrategyHostAssisted = "copy"
+	CloneStrategyHostAssisted CDICloneStrategy = "copy"
```

Comment:
In yaml it is still a string so no problem that we are changing it here, right?

Reply:
Right it is actually still a 'string' in memory, it is just properly types as something besides a string in the code. This way we don't have to do as many casts to the data structure type.